.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: Build the documentation
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    name: Run tests
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.10-dev, 3.9, 3.8, 3.7, 3.6, 3.5, 3.4.10, 2.7]
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r test-requirements.txt
+
+      - name: Run tests
+        env:
+          PYTHON_VERSION: ${matrix.python-version}
```

Isn't the following correct?

```suggestion
          PYTHON_VERSION: ${{ matrix.python-version }}
```